validator_test.go

```diff
@@ -552,24 +559,24 @@ func TestValidateRequiredFailing(t *testing.T) {
 		{ErrRequired, &struct {
 			A int `validate:"required"`
 		}{}},
-		{ErrRequired, &struct {
+		{ErrEmpty, &struct {
```

Comment:
oh, string is special. Just like for `int` I would expect an `ErrRequired` here.